apps/gallery/utils.py

```diff
@@ -0,0 +1,20 @@
+from django.conf import settings
+from django.urls import reverse
+from django.utils.encoding import force_bytes
+from django.utils.http import urlsafe_base64_encode
+
+from gallery.enums import PrivateFileModuleType
+
+
+def get_private_file_url(obj, id):
```

Comment:
Let's change obj to `private_file_module_type: PrivateFileModuleType`

Call this function without .value, the function should do the .value internally